File: dart/beginner/1011.dart
import 'dart:io';
import 'dart:math';

double _sphereArea(int radius) {
  double PI = 3.14159;
  return (4.0 / 3) * PI * pow(radius, 3);
}

void main(List<String> args) {
  int radius = int.parse(stdin.readLineSync().toString().trim());
  double sphereArea = _sphereArea(radius);

  String textSphereArea = sphereArea.toStringAsFixed(3);

  print("VOLUME = $textSphereArea");
}

File: C/beginner/1011.c
#include <stdio.h>
#include <math.h>

int main() {

    const double PI = 3.14159;
    double radius;

    scanf("%lf", &radius);

    double volume = (4.0/3) * PI * pow(radius, 3.0);

    printf("VOLUME = %.3lf\n", volume);

    return 0;
}